src/main/java/ch/njol/skript/expressions/ExprFallenDistance.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * This file is part of Skript.
+ *
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Fall Distance")
+@Description({"The distance an entity has fallen for"})
+@Examples({"set all entities' fall distance to 10",
+	"on damage:",
+	"\tsend \"%victim's fall distance%\" to victim"})
+@Since("INSERT VERSION")
+public class ExprFallenDistance extends SimplePropertyExpression<Entity, Number> {
```

Comment:
Class should be ``ExprFallDistance`` with the file being ``ExprFallDistance.java``

Reply:
Does it really matter?

Reply:
Your ``@Name`` is ``("Fall Distance")``, not ``("Fallen Distance")``

Reply:
I 100% agree.
This was something I had previously reviewed, but it was ignored and "resolved"
<img width="888" alt="Screen Shot 2020-07-20 at 3 12 15 AM" src="https://user-images.githubusercontent.com/20008482/87926668-e2c7a180-ca36-11ea-85d0-db1b545f6abc.png">

Reply:
My bad